application_sdk/constants.py

```diff
@@ -84,6 +86,19 @@
 )
 #: Maximum number of activities that can run concurrently
 MAX_CONCURRENT_ACTIVITIES = int(os.getenv("ATLAN_MAX_CONCURRENT_ACTIVITIES", "5"))
+#: Whether Temporal authentication is enabled
+WORKFLOW_AUTH_ENABLED = (
+    os.getenv("ATLAN_WORKFLOW_AUTH_ENABLED", "false").lower() == "true"
+)
+#: OAuth2 token endpoint URL for Temporal authentication
+WORKFLOW_AUTH_URL = (
+    os.getenv("ATLAN_WORKFLOW_AUTH_URL")
+    or f"https://{WORKFLOW_HOST.replace('-temporal', '').replace('..', '.').replace('--', '-').strip('.-')}/auth/realms/default/protocol/openid-connect/token"
```

Comment:
just keep it simple env, do not infer.

Reply:
Lets make this also part of the deployment secret store